tests/scripts/generate_tls13_compat_tests.py

```diff
@@ -71,10 +70,11 @@
 }
 
 
-class TLSProgram(metaclass=abc.ABCMeta):
+class TLSProgram:
```

Comment:
Why should we remove the "metaclass=abc.ABCMeta". Maybe define it to be one abstract class will have no side effect.